[refactor] Move some class/functions in test_jit.py to jit_utils.py

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #26978 [refactor] Remove unimplmented passes
* **#26839 [refactor] Move some class/functions in test_jit.py to jit_utils.py**

Summary:
att

Test Plan:
ci

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17643010](https://our.internmc.facebook.com/intern/diff/D17643010)

jerryzh168 committed:
[refactor] Move some class/functions in test_jit.py to jit_utils.py

Summary:
att

Test Plan:
ci

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[refactor] Move some class/functions in test_jit.py to jit_utils.py"


Summary:
att

Test Plan:
ci

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[refactor] Move some class/functions in test_jit.py to jit_utils.py"


Summary:
att

Test Plan:
ci

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM commented on test/jit_utils.py:
```diff
@@ -579,3 +579,27 @@ def wrapper(*args, **kwargs):
         return noop_fuser
+
+
+class Observer(torch.nn.Module):
```
I'm not sure about this - it doesn't seem as generic as everything else in this file. It's really a very test specific (even though we have several tests using it). Ideally I think that should live in some other file along with the tests for JIT passes for quantization. If other people think it's fine, I don't mind though.

jerryzh168 replied:
put them in JitTestCase?

ZolotukhinM commented on test/jit_utils.py:
```diff
@@ -579,3 +579,27 @@ def wrapper(*args, **kwargs):
+
+def get_forward(c):
+    return c._get_method('forward')
+def get_forward_graph(c):
```
This part looks good to me.

jerryzh168 committed:
Update on "[refactor] Move some class/functions in test_jit.py to jit_utils.py"


Summary:
att

Test Plan:
ci

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
LGTM!
